AlphaWallet/Common/Services/SpamTokenService.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SpamTokenService.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 3/4/23.
+//
+/// This service subscribes to the AlphaWalletTokensService for add tokens notifications.
+/// When a spam token is detected, this service will hide it.
+
+import Foundation
+import AlphaWalletFoundation
+import Combine
+
+class SpamTokenService {
+
+    private let tokenGroupIdentifier: TokenGroupIdentifierProtocol
+    private var cancellableSet: Set<AnyCancellable> = []
```

Comment:
replace with `private var cancellable = Set<AnyCancellable>()`, most of subscriptions in the app store in `cancellable`